sys-apps/heirloom-tools/files/heirloom-tools-070715-gcc-15.patch

```diff
@@ -0,0 +1,524 @@
+Port to C99/C23. Correct usage of timeval in utmp, rename reserved keywords
```

Comment:
I would've just done -std=gnu17 and fixed the rest but not bothered with the renaming giving how large it makes the patch and it's not upstreamable (given upstream is dead), but fine.